packages/css/src/components/Item.css

```diff
@@ -21,7 +21,7 @@
   cursor: pointer;
 }
 
-.f-Item.has-action:hover {
+.f-Item.has-action:hover, .f-Item.is-focused {
```

Comment:
```suggestion
.f-Item.has-action:hover, .f-Item.has-action:focus {
```
?

Reply:
Can't be, this is for the sake of `react-select` and it's executing focus on element by adding `is-focused`